test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1184,9 +1187,10 @@ void UcxConnection::request_completed(ucx_request *r)
     ucs_list_del(&r->pos);
 
     if (_disconnect_cb != NULL) {
-        UCX_CONN_LOG << "completing " << r->what << " request " << r
-                     << " with status \"" << ucs_status_string(r->status)
-                     << "\" (" << r->status << ")" << " during disconnect";
+        UCX_CONN_LOG << "completing " << r->what << " (length=" << r->length
+                     << ") "" request " << r << " with status \""
+                     << ucs_status_string(r->status) << "\" (" << r->status
```

Comment:
maybe print the length from use callback in io_demo.cc?
it should have more request info such as operation, seed, etc. and should already have the length